[EN/ZH] Update `GBC 2022 Autumn` (#8179)
ppy/osu-wiki: wiki/Tournaments/GBC/GBC_2022_Autumn/en.md

| Mappool selector | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407), ::{ flag=CN }:: [Hylotl](https://osu.ppy.sh/users/18270260), ::{ flag=CN }:: [\[GB\]Luoxuan0327](https://osu.ppy.sh/users/8586018), ::{ flag=CN }:: [YuEast 2018](https://osu.ppy.sh/users/13953619), ::{ flag=CN }:: [\[GB\]Sanae](https://osu.ppy.sh/users/11238501) |
| SV mappool selector | ::{ flag=CN }:: [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561) |
| Mappool helper | ::{ flag=CN }:: [Blue_Potion](https://osu.ppy.sh/users/13094831), ::{ flag=GB }:: [-Deepdive-](https://osu.ppy.sh/users/13338645), ::{ flag=CN }:: [Lubei_01](https://osu.ppy.sh/users/11692528) |
| Referee | *TBA* |
| Commentator | *TBA* |
| Streamer | ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407), ::{ flag=CN }:: [aiyulu](https://osu.ppy.sh/users/189617) |
| Designer | ::{ flag=CN }:: [\[GB\]PlayandyoU](https://osu.ppy.sh/users/14759634), ::{ flag=CN }:: [\[GB\]Mafufu](https://osu.ppy.sh/users/10884561)  |

> | Statistician & Wiki Editor | ::{ flag=CN }:: [\[GB\]Rush\_FTK](https://osu.ppy.sh/users/3046856), ::{ flag=CN }:: [\[GB\]Yuria](https://osu.ppy.sh/users/13092888), ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407), ::{ flag=CN }:: [\[GB\]Natro](https://osu.ppy.sh/users/27233308) |

Reviewer: ```suggestion
| Statistician & Wiki editor | ::{ flag=CN }:: [\[GB\]Rush\_FTK](https://osu.ppy.sh/users/3046856), ::{ flag=CN }:: [\[GB\]Yuria](https://osu.ppy.sh/users/13092888), ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407), ::{ flag=CN }:: [\[GB\]Natro](https://osu.ppy.sh/users/27233308) |
```

> | Statistician & Wiki Editor | ::{ flag=CN }:: [\[GB\]Rush\_FTK](https://osu.ppy.sh/users/3046856), ::{ flag=CN }:: [\[GB\]Yuria](https://osu.ppy.sh/users/13092888), ::{ flag=CN }:: [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407), ::{ flag=CN }:: [\[GB\]Natro](https://osu.ppy.sh/users/27233308) |

Author: Same with TGPGL's suggestion already committed.

### Stage instructions

1. Following the Qualifier stage, all player should participate a phase to form teams , called `Drafting phase`, unpicked player will be eliminated from the tournament.
2. Following the Drafting phase, a swiss stage will be played, called `Regular stage`. Top 16 teams will be advanced, others are eliminated from the tournament.

> 3. Following the Regular stage, a single elimination stage will be played, called `Play-offs`. This means that the winner continues with their runs, and the losing team will be eliminated from the tournament, except in the bronze match.

Reviewer 2: ```suggestion
3. Following the Regular stage, a single-elimination stage will be played, called *Playoffs*. This means that the winner continues with their runs, and the losing team will be eliminated from the tournament, except in the bronze match.
```

   - Fewest beatmaps lost
   - Sum of all score differences of each beatmap (`∑(total score - opponent's total score)` for each beatmap played in the regular stage)
   - Dice roll
3. On Round 4, the highest ranked `2-1` team will faced to the lowest ranked `1-2` team。
4. The top 16 teams will advanced to the Playoff stage, while the remaining players get eliminated from the tournament.

>    - This means that teams with a record of `3-X` will advance to the next stage, while teams with a record of `X-3` will be eliminated.

Reviewer 2: ```suggestion
   - This means that teams with a record of 3-X will advance to the next stage, while teams with a record of X-3 will be eliminated.
```

>    - Also, teams with a record of `2-2` will have chance to advanced by ranking top 16.
>    - Teams with a record `3-0` or `0-3` needn't play round 4, as they are already advanced or get eliminated.

Reviewer 2: ```suggestion
   - Also, teams with a record of 2-2 will have the chance to advance by ranking in the top 16.
   - Teams with a record 3-0 or 0-3 need not play round 4, as they have already advanced or gotten eliminated.
```

### Mappool instructions

> 1. There will be five separate mappools, *this might change depending on the number of incoming registrations.*.

Reviewer 2: ```suggestion
1. There will be five separate mappools. *This might change depending on the number of incoming registrations.*
```

>    - Qualifiers use separate mappool.

Reviewer 2: ```suggestion
   - Qualifiers will use a separate mappool.
```

   - Regular stage Round 1 and Round 2 use mappool \#1.
   - Regular stage Round 3 and Round 4 use mappool \#2.
   - Round of 16 and Quarterfinals use mappool \#3.
   - Semifinals and Finals use mappool \#4.
2. Each mappool consists of a fixed amount of maps each stage which will all be played under [FreeMod](/wiki/Game_modifier#freemod) conditions. This means that there is a unique FreeMod bracket.
3. The mappool sizes are as follows:
   - Qualifiers: 8 beatmaps   
   - Mappool \#1 & \#2: 15 beatmaps
   - Mappool \#3 & \#4: 17 beatmaps.
4. Each mappool has one tiebreaker, except for the Qualifiers.

> 5. Possible mod choices for the FreeMod bracket are NoFail, Hidden, FadeIn, Flashlight, and Mirror.

Reviewer 2: ```suggestion
5. Possible mod choices for the FreeMod bracket are No Fail, Hidden, Fade In, Flashlight, and Mirror.
```
